qep-320-backport-policy.md

```diff
@@ -0,0 +1,56 @@
+# QGIS Enhancement 320: Backport policies
+
+## Summary
+
+This QEP documents the standards and requirements for backporting changes from the ``master`` branch to
+Long Term Release (LTR) and current stable release branches in QGIS. Developers must follow these guidelines
+when submitting backports.
+
+Notes:
+
+- These guidelines describe QGIS specific backporting practices only. Following general
+  "best practice" for software maintenance and version control is assumed.
+- Different policies apply for backports to stable release branches versus LTR branches.
+
+## General Backporting Guidelines
+ 
+- 1.1. Changes must be merged to ``master`` before backporting.
+- 1.2. Backports must be submitted using Pull Requests.
+- 1.3. A backport must be approved by a core QGIS developer prior to merge.
+  - 1.3.1. The developer of the original change is not allowed to "self-approve"
+   their own backported changes.
+- 1.4. Changes which are permissible for backporting are:
+  - 1.4.1. Bug fixes
+  - 1.4.2. API modifications and additions which have a low risk of introducing regressions
+  - 1.4.3. Test case additions
+  - 1.4.4. Documentation fixes and improvements
+  - 1.4.5. Performance optimizations
+- 1.5. Changes which are NOT permissible for backporting include:
+  - 1.5.1 New features
+- 1.6. Try to avoid backporting changes which introduce new strings for translation,
+  or modifications to existing string translations. These usually result in an English
+  language string appearing within QGIS, which is undesirable.
```

Comment:
Does this apply starting from .0?

Reply:
@DelazJ I'm not sure ... Do you think it should?

Reply:
No, not at all. I didn't know this was a rule actually. But I'm probably not the right person to speak on this given that French community (like many other communities however) hasn't reached full translation for years: reading Frenglish is part of our QGIS user experience 😢 
But as an example, I hesitated to ask backport to 3.40 for https://github.com/qgis/QGIS/pull/60479 after I read this advice. I think it should...

Reply:
@jef-n do you have any thoughts here?

Reply:
@DelazJ 

On second thoughts, I think we should leave this as-is. The QEP is intended to be a reflection of **current practices**, which are that backports of translation changes are discouraged. If we want to loosen this policy then we should do so in a separate follow-up proposal.